spec/cancan/ability_spec.rb

```diff
@@ -500,6 +501,25 @@ class Container < Hash
       expect(@ability.unauthorized_message(:update, :missing)).to be_nil
     end
 
+    it "uses model's name in i18n" do
+      class Account
+        include ActiveModel::Model
+      end
+
+      I18n.backend.store_translations :en,
+                                      activemodel: { models: { account: 'english name' } },
+                                      unauthorized: { update: { all: '%{subject}' } }
+      I18n.backend.store_translations :ja,
+                                      activemodel: { models: { account: 'japanese name' } },
+                                      unauthorized: { update: { all: '%{subject}' } }
+
+      I18n.default_locale = :en
+      expect(@ability.unauthorized_message(:update, Account)).to eq('english name')
```

Comment:
could you please use a 
```
I18n.with_locale(:fr) do
end
```

block? this avoids collateral effects and allow us to avoid the line 494

Reply:
@coorasse Thx! I fixed:  https://github.com/CanCanCommunity/cancancan/pull/462/commits/d78398ff04ead814a96a4d97dd179d8b70b29056